app.py

```diff
@@ -23,6 +23,9 @@
 
 cdk_env = cdk.Environment(account=aws_account, region=aws_region)
 
+bde_host = deployment_environment.get("bde_host")
+bde_ro_user = deployment_environment.get("bde_ro_user")
```

Comment:
Do we want to pass these all the way through the call stack? We could just fetch them where they are needed.

Reply:
There is a bit of passing around isn't it. But the tradeoff is that we have all the configuration in one place. Should we need to update some config in the future, we don't need to step through the lambda code... Thoughts?

Reply:
Oh, I'm not arguing for putting the configuration in the code. The configuration can still be in one place, it just could be fetched where it's needed within the CDK code.

Reply:
Have removed parsing `deployment_environment` into the stack unnecessarily, as discussed.